umap/templates/umap/login_popup_end.html

```diff
@@ -2,10 +2,13 @@
 <h3>{% trans "You are logged in. Continuing..." %}</h3>
 <script type="text/javascript">
   function proceed() {
-    if (window.opener && window.opener.umap_proceed) {
-      window.opener.umap_proceed()
+    if (window.opener && window.opener.onLogin) {
```

Comment:
```suggestion
    if (window.?opener.?onLogin) {
```

todo: run biome against html files 😇